src/js/index.js

```diff
@@ -0,0 +1,55 @@
+import Expression from "./Expression.js"
+
+
+const selectElements = (query) => {
+    return document.querySelectorAll(query);
+}
+
+const selectElement = (query) => {
+    return document.querySelector(query);
+}
+
+const setClickEvts = (elements, callback) => {
+    elements.forEach(element => {
+        element.addEventListener('click', callback())
+    });
+}
+
+window.onload = ()=>{
+    const expression = new Expression();
+
+    const total = selectElement('#total');
+    const setTotal = (text) =>{
+        total.innerText=text;
+    }
+
+    const getOperand = () =>{
+        return (event) => {
+            expression.getOperand(event.target.innerText);
+            setTotal()
+        }
+    }
+
+    const getOperator = () => {
+        return (event) => {
+            expression.getOperator(event.target.innerText);
+            const result = expression.getResult();
+            if (result !== null) {
+                document.querySelector('#total').innerHTML = result;
+                setTotal(result)
+                return;
+            }
+
+            setTotal(expression.getExpression())
+        }
+    }
+
+    const digitBtns = selectElements('.digit');
+    const operatorBtns = selectElements('.operator');
+    const clearBtns = selectElements('.modifier');
+
+    setClickEvts(digitBtns, getOperand)
+    setClickEvts(operatorBtns, getOperator)
+    setClickEvts(clearBtns, ()=> expression.clear())
```

Comment:
12-16 라인에서 말씀드린게 아래처럼 부모요소 하나에만 거는 방법입니다.
[이벤트버블링](https://ko.javascript.info/bubbling-and-capturing) 보시면 잘 나와있습니다. 
```suggestion
    const digitBtns = selectElements('.digits');
    const operatorBtns = selectElements('.operations');
    const clearBtns = selectElements('.modifier');

    setClickEvts(digitBtns, getOperand)
    setClickEvts(operatorBtns, getOperator)
    setClickEvts(clearBtns, ()=> expression.clear())
```